<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>retrait</class>
 <widget class="QMainWindow" name="retrait">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#title{
	color: green;
	font-size: 18px;
}</string>
    </property>
    <property name="text">
     <string>RETRAIT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="retirer">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#retirer{
	background-color:  #eeeeee;
    color: green;
}</string>
    </property>
    <property name="text">
     <string>Retirer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="supprimer">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#supprimer{
	background-color:  #eeeeee;
    color: green;
}</string>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modifier">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>320</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#modifier{
	background-color:  #eeeeee;
    color: green;
}</string>
    </property>
    <property name="text">
     <string>Modifier le montant</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="numComptes">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>90</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_comptes">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>60</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Numero de comptes:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>120</y>
      <width>94</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Numero de cheque:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="numCheque">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>150</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>180</y>
      <width>96</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Montant de retrait</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="montant">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>210</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>240</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Date de retrait (yyyy-mm--dd):</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>120</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>180</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>240</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>300</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="listeTitle">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#listeTitle{
	color: green;
	font-size: 18px;
}</string>
    </property>
    <property name="text">
     <string>LISTE DES RETRAITS</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>140</y>
      <width>451</width>
      <height>251</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>360</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>431</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="indication">
       <property name="text">
        <string>numero de compte:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="compteSearch"/>
     </item>
     <item>
      <widget class="QPushButton" name="afficheRetrait">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">#afficheRetrait{
	background-color:  #eeeeee;
    color: green;
}</string>
       </property>
       <property name="text">
        <string>Afficher les retraits de cette compte</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>116</width>
      <height>16</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="nombreRetrait">
       <property name="styleSheet">
        <string notr="true">#nombreRetrait{
	color: green;
	font-size: 12px;
}</string>
       </property>
       <property name="text">
        <string>Nombre de retrait:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="listeNumber">
       <property name="styleSheet">
        <string notr="true">#listeNumber{
	color: red;
}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>420</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#back{
	background-color:  #eeeeee;
    color: green;
}</string>
    </property>
    <property name="text">
     <string>&lt;&lt;Precedent</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="date">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>270</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="actualiser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#actualiser{
	background-color:  #eeeeee;
    color: green;
}</string>
    </property>
    <property name="text">
     <string>Actualiser</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modifierDate">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>320</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#modifierDate{
	background-color:  #eeeeee;
    color: green;
}</string>
    </property>
    <property name="text">
     <string>Modifier la Date</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>955</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
